src/ocp_postprocess/cluster_domain_rename/rename_utils.rs

```diff
@@ -825,3 +825,37 @@ pub(crate) fn fix_kapi_startup_monitor_pod_yaml(pod_yaml: &str, original_hostnam
     let replacement = format!(r"--node-name={}", hostname);
     Ok(pod_yaml.replace(&pattern, &replacement))
 }
+
+pub(crate) fn override_machineconfig_source(machineconfig: &mut Value, new_source: &str, path: &str) -> Result<()> {
```

Comment:
Can you create a `mod` called `utils` under `pull_secret_rename` and move this function there? This `rename_utils` `mod` is under `cluster_domain_rename` and is meant only for domain rename